src/test/java/hw2/base/BaseTest.java

```diff
@@ -0,0 +1,146 @@
+package hw2.base;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+import java.util.List;
+
+import static hw2.base.Const.*;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class BaseTest {
+
+    protected WebDriver driver;
+
+    @BeforeClass
+    public void setUp() {
+        System.setProperty("webdriver.chrome.driver", "src\\main\\resources\\chromedriver.exe");
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+    }
+
+    @AfterClass
+    public void tearDown() {
+        driver.quit();
+    }
+
+    //ex1 and ex2 common steps
+    protected void openURL() {
+        driver.get(BASE_URL);
+    }
+
+    public void assertTitle() {
+        assertEquals(driver.getTitle(), titleExpected);
+    }
+
+    protected void performLogin() {
```

Comment:
Okay, if I wan to login as another user? What I should do?

Please parametrize methods like:
performLogin() - > performLogin(String username,String password)